homeassistant/config_entries.py

```diff
@@ -2761,11 +2761,29 @@ def async_update_reload_and_abort(
         unique_id: str | None | UndefinedType = UNDEFINED,
         title: str | UndefinedType = UNDEFINED,
         data: Mapping[str, Any] | UndefinedType = UNDEFINED,
+        data_updates: dict[str, Any] | UndefinedType = UNDEFINED,
```

Comment:
Shouldn't this be `Mapping`, or do we want to mutate the `data_updates?

Reply:
```suggestion
        data_updates: Mapping[str, Any] | UndefinedType = UNDEFINED,
```